schema/v1/app.cue

```diff
@@ -24,12 +25,17 @@ package v1
 
 #Job: {
 	#ContainerBase
+	#WorkloadBase
 	labels:                       [string]: string
 	annotations:                  [string]: string
 	schedule: string | *""
 	sidecars: [string]: #Sidecar
 }
 
+#WorkloadBase: {
+	workloadClass?: string
```

Comment:
Can this just be `class`, which will line up with `volume: class: "foo"`

Reply:
@thedadams thats how volume is correct?

Reply:
So something like this?

```cue
#WorkloadBase: {
    workload: {
        class?: string
    }
}
```

Reply:
I don't know if it makes sense to do `volume: class: "foo"` because you would have to give the volume a name, but that is the setup for volume classes.

@tylerslaton I don't think we want
```
#WorkloadBase: {
    workload: {
        class?: string
    }
}
```
because that would lead to Acornfiles that look like
```
containers: {
    "web": {
        image: "nginx"
        "web":
            class: <NAME OF CLASS>
    }
}
```

Given the context that this would be placed inside a `container` or `job` block, I think `class` is fine, which would mean Acornfiles would look like
```
containers: {
    "web": {
        image: "nginx"
        class: <NAME OF CLASS>
    }
}
```

Reply:
Ah, I see. Thank you for the clarification.